### ISSUE TITLE
Problem when create png file to save raster

### ISSUE THREAD

**Issue by gustavo234guimaraes** [dime-worldbank/googletraffic#9]
I have identified a small error while using the package.
Not sure why the png file wasn't created before running the "webshot" function. I added a dir.create and file.create command before the webshot and it worked.

**Comment by ramarty** [comment]
@gustavo234guimaraes ah thanks for flagging this!

And just to make sure, is the issue with the `gt_make_raster` function or the `gt_make_png` function? Are you able to show a screenshot of the error you were getting?

Basically the process is (1) create HTML that shows interactive map with traffic --> (2) Webshot creates png from HTML / interactive map --> (3) png converted to raster.

If you're able to point to where you added `dir.create` and `file.create` I can add that to the package to resolve the issue -- or feel feel to make a pull request too!

**Comment by gustavo234guimaraes** [comment]
The error is in gt_html_to_raster.
I identified after the first image error and corrected it as shown in the third image.
I don't have very advanced knowledge in R, so I don't know if the error was actually the lack of these commands, because I've used the package other times and it worked perfectly.

![Screenshot_1](https://user-images.githubusercontent.com/49100132/224093977-ebebab43-1f65-4709-b261-3187c731026a.png)
![Screenshot_2](https://user-images.githubusercontent.com/49100132/224094042-02a0b135-bfe5-49f4-a189-3d5242fbed61.png)
![Screenshot_3](https://user-images.githubusercontent.com/49100132/224094115-80ac4ab4-850f-4509-9a3f-7390237e8014.png)
